doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,327 @@
+determined by evaluating `ENV["JULIA_EDITOR"]`.
+
+The environment variables that Julia uses generally start with `JULIA`. If
+[`Base.versioninfo`](@ref) is called with `verbose` equal to `true`, then the
+output will list all environment variables for which `JULIA` appears in the
+name.
+
+## File locations
+
+### `JULIA_HOME`
+
+The absolute path of the directory containing the Julia executable, which sets
+the global variable [`Base.JULIA_HOME`](@ref). If `$JULIA_HOME` is not set, then
+Julia determines the value `Base.JULIA_HOME` at run-time.
+
+The executable itself is one of
+
+```
+$JULIA_HOME/julia
+$JULIA_HOME/julia-debug
+```
+
+by default.
+
+The global variable `Base.DATAROOTDIR` determines a relative path from
+`Base.JULIA_HOME` to the data directory associated with Julia. Then the path
+
+```
+$JULIA_HOME/$DATAROOTDIR/julia/base
+```
+
+determines the directory in which Julia initially searches for source files (via
+`Base.find_source_file()`).
+
+Likewise, the global variable `Base.SYSCONFDIR` determines a relative path to the
+configuration file directory. Then Julia searches for a `juliarc.jl` file at
+
+```
+$JULIA_HOME/$SYSCONFDIR/julia/juliarc.jl
+$JULIA_HOME/../etc/julia/juliarc.jl
+```
+
+by default (via `Base.load_juliarc()`).
+
+For example, a Linux installation with a Julia executable located at
+`/bin/julia`, a `DATAROOTDIR` of `../share`, and a `SYSCONFDIR` of `../etc` will
+have `JULIA_HOME` set to `/bin`, a source-file search path of
+
+```
+/share/julia/base
+```
+
+and a global configuration search path of
+
+```
+/etc/julia/juliarc.jl
+```
+
+### `JULIA_LOAD_PATH`
+
+A separated list of absolute paths that are to be appended to the variable
+[`LOAD_PATH`](@ref). (In Unix-like systems, the path separator is `:`; in Windows
+systems, the path separator is `;`.) The `LOAD_PATH` variable is where
+[`Base.require`](@ref) and `Base.load_in_path()` look for code; it defaults to the absolute
+paths
+
+```
+$JULIA_HOME/../local/share/julia/site/v$(VERSION.major).$(VERSION.minor)
+$JULIA_HOME/../share/julia/site/v$(VERSION.major).$(VERSION.minor)
+```
+
+so that, e.g., version 0.5 of Julia on a Linux system with a Julia executable at
+`/bin/julia` will have a default `LOAD_PATH` of
+
+```
+/local/share/julia/site/v0.5
+/share/julia/site/v0.5
+```
+
+### `JULIA_PKGDIR`
+
+The path of the parent directory `Pkg.Dir._pkgroot()` for the version-specific
+Julia package repositories. If the path is relative, then it is taken with
+respect to the working directory. If `$JULIA_PKGDIR` is not set, then
+`Pkg.Dir._pkgroot()` defaults to
+
+```
+$HOME/.julia
+```
+
+Then the repository location [`Pkg.dir`](@ref) for a given Julia version is
+
+```
+$JULIA_PKGDIR/v$(VERSION.major).$(VERSION.minor)
+```
+
+For example, for a Linux user whose home directory is `/home/alice`, the directory
+containing the package repositories would by default be
+
+```
+/home/alice/.julia
+```
+
+and the package repository for version 0.5 of Julia would be
```

Comment:
I disagree. It highlights the version being the point why being mentioned as an example, rather than being a specific behaviour _of_ version 0.5